leasing/management/commands/create_invoices.py

```diff
@@ -22,6 +22,7 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):  # noqa: C901 TODO
         override = options.get("override", False)
         today = datetime.date.today()
+        today = today.replace(day=1)
```

Comment:
This stands out a bit, why is this day replacing added?

Reply:
Oops, it was for testing purposes :joy: